odl/space/npy_ntuples.py

```diff
@@ -1387,6 +1387,26 @@ def adjoint(self):
         return MatVecOperator(self.matrix.conj().T,
                               domain=self.range, range=self.domain)
 
+    @property
+    def inverse(self):
+        """Inverse operator represented by the inverse matrix.
+
+        Taking the inverse causes sparse matrices to become dense and is
+        generally very heavy computationally. It is recommended to instead
+        use one of the solvers available in the solvers package.
```

Comment:
Also: since this doc is hard to get to (not reachable in Spyder with Ctrl-I for example), consider mentioning in the class doc.